packages/components/app/styles/components/link/standalone.scss

```diff
@@ -21,7 +21,7 @@ $hds-link-standalone-border-width: 1px;
   align-items: center;
   justify-content: center;
   width: fit-content;
-  font-weight: var(--token-typography-font-weight-medium);
+  font-weight: var(--hds-font-weight-regular);
```

Comment:
Let me know if we would want to change this from specifying the style here to instead using a helper classname.

Reply:
where is the `--hds-font-weight-regular` coming from? 🤔
afaik the token is `--token-typography-font-weight-regular` (maybe you're using the CSS helper class name as token?

Reply:
I had to go look because I was wondering about this same thing. I did not find it under foundations/typography but foundations/tokens:

![CleanShot 2023-09-01 at 13 29 33](https://github.com/hashicorp/design-system/assets/4587451/ed708132-7662-4976-ab9d-efb45436bedb)

I wonder if it would be beneficial to link to that part of the token docs in the typography docs for more clarity?

Reply:
Oops, sorry about that. I will fix

Reply:
Pushed a fix

Reply:
> I wonder if it would be beneficial to link to that part of the token docs in the typography docs for more clarity?

It is linked: https://helios.hashicorp.design/foundations/typography?tab=code#design-tokens (only, is the less preferred/suggested way of using the HDS typography, hence it's not too much in evidence)